ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/Constants.java

```diff
@@ -94,6 +94,8 @@
   public static final int WITHDRAWABLE = 2;
   public static final int MAX_WITHDRAWALS_PER_EPOCH = 4;
 
+  public static final int GENESIS_SLOT = (int) Math.pow(2, 19); //
```

Comment:
This could be 2 << 19. But I don't care a whole lot either way.

Reply:
Leaving this, to conform with the code already in the class